synapse/lib/storm.py

```diff
@@ -606,6 +606,8 @@ def ask(self, text, data=(), timeout=None):
         Run a storm query and return the query result dict.
         user= stdin=
         '''
+        user = s_auth.whoami()
+        logger.debug('Executing storm query [%s] as [%s]', text, user)
```

Comment:
This def needs to be a configurable behavior that defaults to off ( but we should probably turn it on for core00 )

Reply:
That's a super easy tweak to make